devel-common/src/tests_common/test_utils/watcher.py

```diff
@@ -16,10 +16,16 @@
 # under the License.
 from __future__ import annotations
 
-from airflow.decorators import task
 from airflow.exceptions import AirflowException
 from airflow.utils.trigger_rule import TriggerRule
 
+from tests_common.test_utils.version_compat import AIRFLOW_V_3_0_PLUS
+
+if AIRFLOW_V_3_0_PLUS:
+    from airflow.sdk import task
+else:
+    from airflow.decorators import task  # type: ignore[attr-defined,no-redef]
```

Comment:
Why would we need to check AIRFLOW_V_3_0_PLUS in main branch?
tests are always running on the latest devel version

Reply:
But when we migrate specific part to SDK then we migrate also the imports. don't we?

Reply:
We do, its been done in https://github.com/apache/airflow/pull/53629. This was a case that got missed

Reply:
And the problem is that its used in provider sys tests.